src/vellum/workflows/nodes/displayable/subworkflow_deployment_node/node.py

```diff
@@ -98,7 +101,9 @@ def run(self) -> Iterator[BaseOutput]:
             event_types=["WORKFLOW"],
             metadata=self.metadata,
             request_options=self.request_options,
+            execution_context={"parent_context": parent_context},
         )
+        # for some reason execution context isn't showing as an option? ^ failing mypy
```

Comment:
[blocking] this is expected since we purposefully omitted it from showing up in our api clients.

instead, we'll need to fill in `request_options.additional_body_parameters` to include the extra body parameter

Reply:
bless 💯